src/Compilers/Core/Portable/Text/TextChange.cs

```diff
@@ -93,5 +94,16 @@ public override int GetHashCode()
         /// An empty set of changes.
         /// </summary>
         public static IReadOnlyList<TextChange> NoChanges => SpecializedCollections.EmptyReadOnlyList<TextChange>();
+
+        internal string GetDebuggerDisplay()
```

Comment:
>internal [](start = 8, length = 8)

nit: is this `internal` instead of `private` because of the unittest?

Reply:
Yes, it helps us generate the test case when the fuzzer finds a failure.